test/e2e/framework/auth/helpers.go

```diff
@@ -122,10 +130,7 @@ func bindInNamespace(c v1beta1rbac.RoleBindingsGetter, roleType, role, ns string
 		Subjects: subjects,
 	})
 
-	// if we failed, don't fail the entire test because it may still work. RBAC may simply be disabled.
-	if err != nil {
-		klog.Errorf("Error binding %s/%s into %q for %v\n", roleType, role, ns, subjects)
-	}
+	return fmt.Errorf("error binding %s/%s into %q for %v", roleType, role, ns, subjects)
```

Comment:
We've filtered out the RBAC disabled case; now it is a real error we should return to the caller.